.github/workflows/build.yml

```diff
@@ -0,0 +1,97 @@
+name: Build, push and deploy faucet image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - alephzero
+    paths-ignore:
+      - ".github/**"
+      - "*.md"
+
+jobs:
+  deploy-faucet:
+    name: Build and push faucet image
+    runs-on: ubuntu-latest
+    steps:
+      - name: GIT | Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: GIT | Checkout Source code
+        uses: actions/checkout@v2
+
+      - name: GIT | Get branch info & current commit sha.
+        id: vars
+        shell: bash
+        run: |
+          echo "##[set-output name=branch;]$(echo ${GITHUB_REF##*/})"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@13d241b293754004c80624b5567555c4a39ffbe3
```

Comment:
Why version of this is particular commit instead of tag like `v1`?

Reply:
This will be for every commit in alephzero repo. So classic workflow, first short commit, then versions for testnet release. Not implemented atm, I'm focusing on devnet for now.

Reply:
No no, I meant `32nd` line:
`uses: aws-actions/configure-aws-credentials@13d241b293754004c80624b5567555c4a39ffbe3`

Reply:
oh, copy pasted from aleph-wallet to be honest. I'll change it